department-of-veterans-affairs/caseflow
# mbeard/APPEALS-32391

Resolves [mbeard/APPEALS-32391](https://jira.devops.va.gov/browse/APPEALS-32391)

# Description
Created StaticLeversWrapper.jsx component which utilize the passed in props with the static array of levers to render the static lever component the appropriate amount of times. 

## Acceptance Criteria

## Testing Plan

1. Go to [Jira Issue/Test Plan Link](https://jira.devops.va.gov/browse/JIRA-32391) or list them below
2. Start storybook and select the story "Static Levers Wrapper"
3. Ensure that the component is rendered correctly and styling is acceptable
4. Check Lighthouse and ANDI for compliance
5. Run Jest Test for 'Static Levers Wrapper' to ensure passing


# Frontend
## User Facing Changes

 BEFORE|AFTER
 ---|---

## Storybook Story
*For Frontend (Presentation) Components*

# Backend
## Database Changes
*Only for Schema Changes*


## Integrations: Adding endpoints for external APIs

# Best practices
## Code Documentation Updates

## Tests
### Test Coverage
Did you include any test coverage for your code? Check below:

### Code Climate
Your code does not add any new code climate offenses? If so why?

## Monitoring, Logging, Auditing, Error, and Exception Handling Checklist

### Monitoring

### Logging

### Auditing

### Error Handling

### Exception Handling
